File: Tugas B/python/memory.py
class memory():
    __freq = ""
    __size = ""
    __cuda = ""

    # construct
    def __init__(self, freq = "", size = "", cuda = ""):
        self.__freq = freq
        self.__size = size
        self.__cuda = cuda

    # set dan get
    def setfreq(self, freq):
        self.__freq = freq

    def getfreq(self):
        return self.__freq
    
    def setsize(self, size):
        self.__size = size
    
    def getsize(self):
        return self.__size

    def setcuda(self, cuda):
        self.__cuda = cuda
    
    def getcuda(self):
        return self.__cuda

    # output
    def printmemory(self):
        print("Freq : " + str(self.__freq))
        print("Size : " + str(self.__size))
        print("Cuda : " + str(self.__cuda))

File: Tugas A/memory.java
public class memory{
    private String freq;
    private String size;
    private String cuda;

    // construct
    public memory() {}

    public memory(String freq, String size, String cuda){
        this.freq = freq;
        this.size = size;
        this.cuda = cuda;
    }
	
	//set and get
	public void setfreq(String freq){
		this.freq = freq;
	}
	public String getfreq(){
		return this.freq;
	}
	
	public void setsize(String size){
		this.size = size;
	}
	public String getsize(){
		return this.size;
	}
	
	public void setcuda(String cuda){
		this.cuda = cuda;
	}
	public String getcuda(){
		return this.cuda;
	}

}